.github/workflows/ci.yaml

```
@@ -89,6 +89,138 @@ jobs:
+          path: .lighthouseci/
+          if-no-files-found: warn
+          retention-days: 7
+
+      - name: Build Lighthouse PR comment body
+        id: lighthouse_comment
+        if: >-
+          always() &&
+          github.event_name == 'pull_request' &&
+          github.event.pull_request.head.repo.full_name == github.repository
+        env:
+          LIGHTHOUSE_MANIFEST: ${{ steps.lighthouse.outputs.manifest }}
```

actionlint (1.7.12)

[error] property "manifest" is not defined in object type {assertionresults: string; links: string; resultspath: string}

(expression)